Lib/python/pyrun.swg

```diff
@@ -269,54 +273,64 @@ SWIG_Python_ExceptionType(swig_type_info *desc) {
 SWIGRUNTIME SwigPyClientData * 
 SwigPyClientData_New(PyObject* obj)
 {
+  SwigPyClientData* data;
```

Comment:
From this point below in this function, there is too much unnecessary whitespace change and unnecessary code refactoring for me to follow what has actually changed in a very core function. Please revert unnecessary changes and if necessary follow up with a whitespace only / no op commit.

Reply:
I had removed a level of indentation, and the github UI is handling this particularly badly. I usually review diffs using Beyond Compare and my local git client \(using TortoiseGit to launch BC\) and as you can see it looks a lot more comprehensible when viewed in BC!

![difference shown in Beyond Compare](https://user-images.githubusercontent.com/32688592/118049350-8642c380-b375-11eb-90d6-dddeb89a759c.png)

However, that said I have looked at the code and I think I can rework to avoid needing to remove the indentation, so will do this so that there's a better diff on github - thanks